src/interface/updatecharactersheet.cpp

```diff
@@ -1576,13 +1581,6 @@ void statsHoverText(Stat* tmpStat)
 							{
 								tmp = players[clientnum]->entity;
 								real_t regen = (static_cast<real_t>(tmp->getHealthRegenInterval(*tmpStat)) / TICKS_PER_SECOND);
-								if ( tmpStat->type == SKELETON )
```

Comment:
Curious why this code got deleted? may need to pull this file again.. this is to show the regen values of a player skeleton.